features/networking/ovn-egress-ip.feature

```diff
@@ -870,3 +870,46 @@ Feature: OVN Egress IP related features
     Then the step should succeed 
     And the output should not contain:
       | name=egressip |
+
+  # @author huirwang@redhat.com
+  # @case_id OCP-42925
+  @admin
+  @destructive
+  @4.10 @4.9
+  @vsphere-ipi
```

Comment:
maybe need `@network-ovnkubernetes` also for CI

Reply:
If add @network-ovnkubernetes, will it be selected by other platforms?

Reply:
this should be for new Prow CI
@liangxia could you help explain more for the tags added recently?

Reply:
You can refer to https://github.com/openshift/verification-tests/blob/master/doc/tags.adoc

Reply:
Thanks for the info.